src/main/resources/static/static/js/11.ba2e73c01467fa9ad6a9.js

```diff
@@ -0,0 +1 @@
a:t})},e.deleteCode=function(t,e){var r=(0,u.reviseParam)(t,e);return(0,o.deleted)({url:i+"contract/"+r.str,method:"delete"})},e.getDeployStatus=function(t){return(0,o.post)({url:i+"contract/deploy",method:"post",data:t,responseType:"text"})},e.sendTransation=function(t){return(0,o.post)({url:i+"trans/handle",method:"post",data:t})},e.ifChangedDepaloy=function(t,e){return(0,o.post)({url:i+"contract/ifChanged/"+t+"/"+e,method:"get"})},e.queryJavaClass=function(t,e){return(0,o.post)({url:i+"contract/compile-java",method:"post",data:t,responseType:"blob/json"})},e.addFunctionAbi=function(t){return(0,o.post)({url:i+"method/add",method:"post",data:t,headers:{Authorization:"Token "+localStorage.getItem("token")||!1}})},e.getFunctionAbi=function(t,e){var r=(0,u.reviseParam)(t,e);return(0,o.get)({url:i+"method/findById/"+r.str,method:"get",headers:{Authorization:"Token "+localStorage.getItem("token")||!1}})},e.addContractpath=function(t){return(0,o.post)({url:i+"contract/addContractPath",method:"post",data:t,headers:{Authorization:"Token "+localStorage.getItem("token")||!1}})},e.performanceSwitch=function(){return(0,o.get)({url:i+"performance/toggle",method:"get",headers:{Authorization:"Token "+localStorage.getItem("token")||!1}})},e.postPerformanceSwitch=function(t){return(0,o.post)({url:i+"performance/toggle",method:"post",data:t,headers:{Authorization:"Token "+localStorage.getItem("token")||!1}})},e.backgroundCompile=function(t){return(0,o.post)({url:i+"contract/contractCompile ",method:"post",data:t})},e.encryption=function(){return(0,o.get)({url:url.ORG_LIST+"/encrypt",method:"get",responseType:"text"})};var n,o=r("rbW/"),u=r("DgvE"),a=r("/tFb"),s=(n=a)&&n.__esModule?n:{default:n};var i=null;i=""},"rbW/":function(t,e,r){"use strict";Object.defineProperty(e,"__esModule",{value:!0});var n=a(r("Asgo"));e.post=function(t){return new n.default((function(e,r){s(t).then((function(t){e(t)})).catch((function(t){r(t)}))}))},e.get=function(t){return new n.default((function(e,r){s(t).then((function(t){e(t)})).catch((function(t){r(t)}))}))},e.patch=function(t){return new n.default((function(e,r){s(t).then((function(t){e(t)})).catch((function(t){r(t)}))}))},e.put=function(t){return new n.default((function(e,r){s(t).then((function(t){e(t)})).catch((function(t){r(t)}))}))},e.deleted=function(t){return new n.default((function(e,r){s(t).then((function(t){e(t)})).catch((function(t){r(t)}))}))};var o=a(r("91MD")),u=a(r("oYx3"));function a(t){return t&&t.__esModule?t:{default:t}}var s=o.default.create({timeout:3e4});s.defaults.headers.post["X-Requested-With"]="XMLHttpRequest",s.defaults.responseType="json",s.defaults.validateStatus=function(){return!0},s.interceptors.request.use((function(t){return t}),(function(t){return n.default.reject(err)})),s.interceptors.response.use((function(t){return t.data&&302e3===t.data.code&&u.default.push({path:"/",query:{redirect:u.default.currentRoute.fullPath}}),t}),(function(t){return n.default.reject(t)})),e.default={axiosIns:s}}}]);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [A constructor name should not start with a lowercase letter.](https://app.codacy.com/gh/WeBankFinTech/WeBASE-Front/pullRequest?prid=4823531)